.travis.yml

```diff
@@ -1,15 +1,15 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
You are right. There was no need in traits. I will fix it. All controllers will extend new abstract class FOS\UserBundle\Controller\ContainerAwareController